### DISCUSSION TITLE
Pop back after grabbing

### DISCUSSION THREAD

**Discussion by trev-dev** [meow-edit/meow#300] [Ideas]
I just realized that when I do something crazy like "select a whole buffer" and grab it to perform some beacon operation, I have to find what I meant to change all over again.

Is it a reasonable idea to suggest that we remember the cursor's last position after an "inside-of-thing" or even any other motion, then when we grab, we pop back to where we started that motion from?

This way when I'm trying to magically change 40 lousy alt="" attributes in an old HTML file, I don't have to re-find my starting point.

**Comment by DogLooksGood** [comment]
I think it could be a bug. 

select, grab, pop, pop should take you back to the original position. 
Current once the selection is canceled, (grab will cancel the selection) the selection history is flushed. I think it's better to keep it.
The downside is you will need two commands for pop grab and pop selection.

**Comment by trev-dev** [comment]
tianshu ***@***.***> writes:

> I think it could be a bug. 
>
> select, grab, pop, pop should take you back to the original position. 
>

I see. When I get a chance I will attempt to reproduce this with a clean
environment. That is, assuming you are not able to reproduce this?

> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/meow-edit/meow/discussions/300#discussioncomment-3418147
> You are receiving this because you authored the thread.
>
> Message ID: ***@***.***>

-- 

Trev : 0FB7 D06B 4A2A F07E AD5B  1169 183B 6306 8AA1 D206

**Reply by DogLooksGood** [reply]
Ah, sorry. I think I misunderstood your idea. Well, now it's not supported. 
Your original position before motion is not saved at the moment.
